src/local-storage.ts

```diff
@@ -17,6 +21,13 @@ export class EppoLocalStorage {
     return null;
   }
 
+  public getKeys(): string[] {
+    if (hasWindowLocalStorage()) {
+      return Object.keys(window.localStorage);
```

Comment:
Is this gonna get EVERYTHING in local storage? I wonder if we should isolate these to all be under some Eppo local storage property.